src/audio_call.c

```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
     print_err (self, error_str);
 }
 
+void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[MAX_STR_SIZE])
+{
+    char *error_str;
+
+    if ( argc != 1 ) {
+        error_str = "Must have value!";
+        goto on_error;
+    }
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = atoi(argv[1]);
```

Comment:
I missed this before but you shouldn't be using atoi(). Use strtol() instead, and be sure to error check it. There are lots of examples you can copy in *_commands.c